common/Perf/Azure.Test.Perf/BatchPerfTest.cs

```diff
@@ -48,9 +48,10 @@ public BatchPerfTest(TOptions options) : base(options)
                 }
                 else
                 {
+#if NETFRAMEWORK
                     // Assume _transport is HttpWebRequestTransport (currently internal class)
                     ServicePointManager.ServerCertificateValidationCallback = (message, cert, chain, errors) => true;
```

Comment:
[SYSLIB0014](https://learn.microsoft.com/en-us/dotnet/fundamentals/syslib-diagnostics/syslib0014): WebRequest, HttpWebRequest, ServicePoint, WebClient are obsolete starting in .NET 6

[`HttpWebRequestTransport`](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/Pipeline/HttpWebRequestTransport.cs) is for NETFRAMEWORK only